app/views/flats/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <button onclick="history.back()">Index</button>
-<h2>Flat Details</p>
-<h2><%= @flat.address %></h2>
-<p><%= @flat.price %></p>
-<p><%= @flat.description %></p>
-<p><%= @flat.city %></p>
+<h2>Restaurant Details</h2>
```

Comment:
Remember we are working with Flats, but great job on the rest and the search bar.